components/fhir-common/src/main/java/org/hl7/gravity/refimpl/sdohexchange/util/FhirUtil.java

```diff
@@ -75,13 +75,27 @@ public LocalDateTime toLocalDateTime(DateTimeType dateType) {
 
   /**
    * Find {@link Coding} by system among a list of {@link CodeableConcept} instances. This is useful when searching for
-   * a specific code inside a FHIR resource.
+   * a specific system inside a FHIR resource.
    */
   public Coding findCoding(List<CodeableConcept> codes, String system) {
+    return findCoding(codes, system, null);
+  }
+
+  /**
+   * Find {@link Coding} by system and code among a list of {@link CodeableConcept} instances. This is useful when
+   * searching for a specific system and code inside a FHIR resource.
```

Comment:
`This is useful when searching for a specific system and code inside a FHIR resource.` and this. The first sentence says it all.